configs/project-refs.conf

```diff
@@ -59,7 +59,10 @@ vars.uris: {
   play-ws-uri:                  "https://github.com/playframework/play-ws.git#8011fb8"  # was master
   pprint-uri:                   "https://github.com/lihaoyi/pprint.git"
   sbinary-uri:                  "https://github.com/scalacommunitybuild/sbinary.git#community-build-2.12"
-  sbt-testng-uri:               "https://github.com/scalacommunitybuild/sbt-testng.git#community-build-2.12"
+  sbt-uri:                      "https://github.com/sbt/sbt.git#1.0.0"
+  sbt-io-uri:                   "https://github.com/sbt/io.git#v1.0.0-M11"
+  sbt-librarymanagement-uri:    "https://github.com/sbt/librarymanagement.git#v1.0.0-X15"
+  sbt-util-uri:                 "https://github.com/sbt/util.git#v1.0.0-M24"
```

Comment:
Thanks Seth, after the unrelated breakage in https://github.com/typesafehub/migration-manager/issues/191 I was thinking about how our cross-repo source/binary breakages in sbt are over so we should get our code building with everything else. (and then I forgot to do anything about it)

The possible targets for compilation are either:

+ the latest tags (RC3 across the board, rather than the misc X??/M?? tags)
+ the temporary 1.0.0 release branches, where the latest RC fixes are being merged, in prep for the v1.0.0 releases, then will be merged into 1.x and deleted
+ the 1.x branches

Personally I'd track 1.0.0, and then switch when we delete them - I'll try and remember to PR or at least ping you.

WRT dbuild, work is progressing I see in https://github.com/typesafehub/dbuild/pull/198, but @cunei's on a well deserved break atm.